Sources/Shared/Extensions/URLExtensions.swift

```diff
@@ -149,6 +149,10 @@ extension URL {
     if let internalUrl = InternalURL(self), internalUrl.isSessionRestore || internalUrl.isWeb3URL {
       return internalUrl.extractedUrlParam?.displayURL
     }
+    
+    if let internalUrl = InternalURL(self), internalUrl.isBlockedPage {
```

Comment:
You can add this condition to the one above. `internalUrl.isSessionRestore || internalUrl.isWeb3URL || internalUrl.isBlockedPage`